src/nv_midbox_websock/templates/.bowerrc

```diff
@@ -0,0 +1,3 @@
+{
+  "directory": "static/bower_components"
```

Comment:
. file?

Do you really think we need to use the bower for packaging? if there is only few file. I would suggest to get rid of the package manager and keep the file in the source tree statically.